plugins/outputs/elasticsearch/elasticsearch.go

```diff
@@ -13,8 +14,6 @@ import (
 
 	"crypto/sha256"
 
-	"gopkg.in/olivere/elastic.v5"
-
```

Comment:
Please move the `"github.com/olivere/elastic"` line to where `"gopkg.in/olivere/elastic.v5"` was before. See my comment in #10513.

Reply:
ah, makes sense, my ide auto added it and collapsed the imports..  my bad